json/src/lib.rs

```diff
@@ -544,6 +550,20 @@ pub struct TestMempoolAcceptResult {
     pub allowed: bool,
     #[serde(rename = "reject-reason")]
     pub reject_reason: Option<String>,
+    /// Virtual transaction size as defined in BIP 141 (only present when 'allowed' is true)
+    /// Added in Bitcoin Core v0.21
+    pub vsize: Option<u64>,
+    /// Transaction fees (only present if 'allowed' is true)
+    /// Added in Bitcoin Core v0.21
+    pub fees: Option<TestMempoolAcceptResultFees>,
+}
+
+#[derive(Clone, PartialEq, Eq, Debug, Deserialize, Serialize)]
+pub struct TestMempoolAcceptResultFees {
```

Comment:
nit: I usually put definitions before their usages, but not gonna hold this up for that :)